server.go

```diff
@@ -5556,6 +5556,16 @@ func (s *server) AttemptRBFCloseUpdate(ctx context.Context,
 	return updates, nil
 }
 
+// needsTimestampIncrement determines whether we need to increment the
+// timestamp for a node announcement to ensure it's at least one second after
+// the previously persisted timestamp. We compare at second precision to match
+// how timestamps are stored in the database (both SQL and KV stores truncate
+// to second precision). This ensures BOLT-07 compliance, which requires node
```

Comment:
IIUC, the reason to do this is not how they are stored but the requirements of BOLT-07. Even if they were stored with sub-second accuracy, the same checks would be needed.

I propose to exclude the following sentence from the godoc: "We compare at second precision to match how timestamps are stored in the database (both SQL and KV stores truncate to second recision)."